test/integration/unlinkFromNumber.test.js

```diff
@@ -0,0 +1,53 @@
+const unlink = require("../../lib/unlinkFromNumber")
+const changeNumber = require("../../lib/changeBotPhoneNumber")
+const twilio = require("twilio")
+
+
+test("Free deployed bot", async () => {
+    let client = twilio("AC16497ae92be880bf536ddf0d8ae92add", "9e9acd8216a57baac83559ee8df1ee00")
+    let resp = await unlink.unlinkFromPhoneNumber(client, "UAa556f1a138444cf5359020bba46f017e")
+    expect(resp).toBe(true)
+    await changeNumber.changeBotPhoneNumber(client, "UAa556f1a138444cf5359020bba46f017e", "+44778260206")
+    })
+
+test("Attempt to free undeployed assistant", async () => {
+    let client = twilio("AC16497ae92be880bf536ddf0d8ae92add", "9e9acd8216a57baac83559ee8df1ee00")
+    let resp = await unlink.unlinkFromPhoneNumber(client, "")
+    expect(resp).toBe(false)
+    })
+
+test("Attempt to free null assistant", async () => {
+    let client = twilio("AC16497ae92be880bf536ddf0d8ae92add", "9e9acd8216a57baac83559ee8df1ee00")
+    let resp = await unlink.unlinkFromPhoneNumber(client, null)
```

Comment:
code duplication. Usually you can do all these 3 tests (where only theinput param differs) as one test by looping through inputs. some testing libs make it easier to define.

Reply:
also you keep building the client variable. usually there are testing hooks such as "beforeEach" which help you aovid that duplication.